lib/std/zig/llvm/bitcode_writer.zig

```diff
@@ -405,6 +406,10 @@ fn charTo6Bit(c: u8) u8 {
     };
 }
 
+fn endianSwap(value: u32) u32 {
+    return if (comptime builtin.cpu.arch.endian() == .big) @byteSwap(value) else value;
+}
```

Comment:
you've reimplemented `std.mem.nativeToLittle`

Reply:
Oh - I searched for "swap" and didn't find anything.